modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -598,24 +592,23 @@ definition {
 	@description = "LPS-161923 - Verify that it's possible to search for notifications"
 	@priority = 3
 	test CanSearchForNotifications {
-		task ("Go to Templates and add Notification Templates") {
+		task ("Given a Notification Templates") {
```

Comment:
Changing the wording shouldn't be part of this commit because it's not part of switching to the API macro